__tests__/api.test.js

```diff
@@ -37,3 +38,24 @@ test("404: will respond with a 404 when the server cannot find the requested res
       expect(body.msg).toBe("Route not found");
     });
 });
+
+describe("GET /api", () => {
+  test("200: GET /api responds with a JSON object describing all available endpoints on the API and how to interact with them", () => {
+    return request(app)
+      .get("/api")
+      .expect(200)
+      .then((res) => {
+        const endpoints = res.body.endpoints;
+        expect(endpoints).toEqual(endpointsJson);
+      });
+  });
+  test("404: will respond with a 404 when the server cannot find the requested resource", () => {
```

Comment:
I'd say this test doesn't just relate to the /api endpoint.  It's a general test for any invalid endpoint.